docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
+:::
 
-If an event emitted by an event source triggers a single workflow, that will count as **two** invocations: one for the source, and one for the workflow. In other words, source and workflow execution is distinct: each counts invocations on its own.
 
-Your workflow's [memory settings](/workflows/settings/#memory) also impact the number of invocations you're charged for each workflow execution. [Read more here](#how-does-workflow-memory-affect-billable-invocations).
+#### Source Credit Usage
 
-#### Scenarios
+The first credit used during a [source's](/docs/sources/) execution is included for free.
+
+This means that the first {{ $site.themeConfig.base_credits_price.seconds }} of computation on an source do not incur credits. This behavior includes [Free Tier](/pricing/#free-tier) accounts.
+
+::: tip
 
-::: details Webhook triggered workflow
+This free first credit per execution **only** applies to sources from the [Pipedream public registry](/docs/sources). 
 
-*1* invocation is incurred per HTTP webhook. The HTTP endpoint is *not* considered a source.
+If you deploy a private custom source to your account, then all computation time including the inital {{ $site.themeConfig.base_credits_price.seconds }} seconds for that private source counted toward credits.
 
 :::
 
-::: details Scheduled workflow
+::: details A polling source finishing under {{ $site.themeConfig.base_credits_price.seconds }} seconds per execution
 
-*1* invocation is incurred per new event emitted by the source.
-*1* invocation is incurred per new event processed by the workflow.
+For example, a source that polls an API for new events like [Airtable - New Row Added](https://pipedream.com/apps/airtable/triggers/new-records) only takes ~5 seconds to poll and emit events to subscribing workflows.
 
-Example: a schedule triggered workflow is configured to run every 15 minutes. Two invocations are incurred every 15 minutes. One from the timer source emitting an event, and the other from the workflow processing the event.
+This would result 0 credits. Regardless of the number of times the source polled because:
 
-```
-2 invocations * 15 minutes * 4 times per hour * 24 hours in a day = 2,880 daily invocations
-```
+* The first {{ $site.themeConfig.base_credits_price.seconds }} of computation time per source execution is included.
+* The **Airtable - New Row Added** source is a [publicly available component](https://pipedream.com/apps/airtable/triggers/new-records).
 
 :::
 
-::: details App webhook powered source triggered workflow
+::: details A polling source finishing over {{ $site.themeConfig.base_credits_price.seconds }} seconds per execution
 
-*1* invocation is incurred per new event emitted by the source.
-*1* invocation is incurred per new event processed by the workflow.
+Consider a long running source that polls a large dataset like an new **RSS - New Item in Feed** that takes 60 seconds total to finish polling per execution.
```

Comment:
You can shorten to:

> Consider an a source that takes 60 seconds total to finish polling, per execution.

Reply:
The example is necessary for the 2nd bullet point that points out this first free credit rule only applies to public registry sources. So I'm going to tighten it up a bit, but still need to reference a public source.